ansible/roles/global_handlers/handlers/main.yml

```diff
@@ -13,6 +13,8 @@
 
 - import_tasks: 'dhcp_probe.yml'
 
+- import_tasks: 'dhcpd.yml'
+
```

Comment:
Don't forget to import the `global_handlers` role in the `dhcpd` role tasks.

Reply:
Crap, our GitLab CI runner actually caught this. I made this PR at the end of the day, didn't want to wait for GitLab CI to report on its status :/